tests/quantization/ggml/test_ggml.py

```diff
@@ -44,6 +44,7 @@ class GgufIntegrationTests(unittest.TestCase):
     phi3_model_id = "microsoft/Phi-3-mini-4k-instruct-gguf"
     bloom_model_id = "afrideva/bloom-560m-GGUF"
     original_bloom_model_id = "bigscience/bloom-560m"
+    stablelm_model_id = "afrideva/stablelm-3b-4e1t-GGUF"
```

Comment:
just like bloom, can you also add a fp16 test to see if we get the same result with the transformers model and the converted gguf model ?

Reply:
Yes, agree, I just added